site/config-overrides.md

```diff
@@ -179,15 +179,15 @@ The following `config.xml` override file demonstrates setting the 'max-message-s
 </domain>
 ```
 
-### Overriding a DataSource Module
+### Overriding a data source module
 
-The following `jdbc-testDS.xml` override template demonstrates setting the URL, username, and password-encrypted fields of a JDBC module named `testDS` via secret macros.  The generated situational config that replaces the macros with secret values will be located in the DOMAIN_HOME/optconfig/jdbc directory.   The `password-encrypted` field will be populated with an encrypted value since it uses a secret macro with a `:encrypted` suffix.  The secret is named `dbsecret` and contains three keys: `url`, `username`, and `password`.
+The following `jdbc-testDS.xml` override template demonstrates setting the URL, user name, and password-encrypted fields of a JDBC module named `testDS` via secret macros.  The generated situational configuration that replaces the macros with secret values will be located in the `DOMAIN_HOME/optconfig/jdbc` directory.   The `password-encrypted` field will be populated with an encrypted value because it uses a secret macro with an `:encrypted` suffix.  The secret is named `dbsecret` and contains three keys: `url`, `username`, and `password`.
```

Comment:
with an encrypted value because it uses a secret macro with an :encrypted suffix. 
should be 
"with an encrypted value because it uses a secret macro with an :**encrypt** suffix."

Reply:
Good catch.  `:encrypted` should be `:encrypt`.